Created unordered list of images with text #20

Blaine137 opened the pull request.

Blaine137 committed:
started to create the organism #20

Blaine137 committed:
updated the unordered list to be more dynamic #20

Blaine137 committed:
removed unused code and updated circle image to align the same as the icons #20

Blaine137 committed:
completed the organism #20

transformerjnm reviewed (changes requested):
Remove unused imports.

Add whitespace between imports and component level comment.

Add a key attribute to each list item in the forEach. Do not use the array index as the key. 

Remove the paper and its styles from the component. The paper makes it look like it's elevated, and Facebook doesn't have that. By removing the paper you will no longer need any of the styles. The width does not need to be set. The component should take up all of the space available and the parent component will handle sizing through a grid.

Blaine137 committed:
implemented requested changes #20

transformerjnm reviewed (commented):
Use let and const instead of var for variable declarations

Do not need the background color for container styles. It will get the proper color once the application background color is set. Unneeded code.

Blaine137 committed:
changed variable declarations, removed color styles from container, and added key to each list item #20

Blaine137 committed:
removed a console.log

transformerjnm committed:
Updated comment and removed unused import for UnorderedListOfIamgesWithText. #20

Removed unused import, updated the comment props section, and passed the action prop to the list item.
